src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -1895,8 +1896,8 @@ protected function fireEvent($input = [])
      */
     public function getRepeaterList()
     {
-        return app(BlockCollection::class)->getRepeaterList()->mapWithKeys(function ($repeater) {
-            return [$repeater['name'] => $repeater];
+        return app(BlockCollection::class)->getRepeaterList()->mapWithKeys(function (Block $repeater) {
+            return [$repeater->name => $repeater->toList()];
```

Comment:
I'm not sure I understand this change. Was the list of repeaters serialized automatically without this and this makes sure we control what ends up in the view through the the `toJson()` above in this file?

Reply:
Before this the $repeaters and $blocks were arrays, I changed this to pass around the Block objects instead.